roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -14,6 +14,21 @@ assetConfig:
 {% if 'metrics_public_url' in openshift.master %}
   metricsPublicURL: {{ openshift.master.metrics_public_url }}
 {% endif %}
+{% if 'extension_scripts' in openshift.master %}
+  extensionScripts:
+{{ openshift.master.extension_scripts | to_padded_yaml(1, 2) }}
```

Comment:
Moving this up to the previous line would prevent adding an additional blank line when openshift.master.extension_scripts is set:
```
  extensionScripts: {{ openshift.master.extension_scripts | to_padded_yaml(1, 2) }}
```